<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_book</class>
 <widget class="QWidget" name="Add_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加图书</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/backgroud.png);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>101</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QLineEdit" name="in_title">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>121</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Title_info">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;书名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_writer">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>161</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="wtiter_info">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;作者&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_ISBN">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>201</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="ISBN_info">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;索书号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_price">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>241</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="price_info">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;价格&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_info">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;数目&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_num">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>281</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sure">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
